cli/cmd/runtime/start.go

```diff
@@ -83,6 +83,9 @@ type Config struct {
 	// AllowHostAccess controls whether instance can use host credentials and
 	// local_file sources can access directory outside repo
 	AllowHostAccess bool `default:"false" split_words:"true"`
+	// DataDir stores data for all instances like duckdb file, temporary downloaded file etc.
+	// The data for each instance is stored in a child directory named instance_id
+	DataDir string `split_words:"true"`
```

Comment:
1. Needs to be added to the shared `.env` file (`rill devtool dotenv upload cloud`). Should probably point to `dev-cloud-state`.
2. Need to remember to get infra to set this on the runtimes. They also need to think apply it to the Kubernetes provisioned runtimes.

Reply:
Ack